stable/insights-agent/templates/opa/rbac.yaml

```diff
@@ -5,27 +5,71 @@ metadata:
   name: {{ include "insights-agent.fullname" . }}-opa
   labels:
     app: insights-agent
-{{- with .Values.opa.additionalAccess }}
+{{- if .Values.opa.defaultTargetResources }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ include "insights-agent.fullname" $ }}-opa-additional
+  name: {{ include "insights-agent.fullname" $ }}-opa-deftargetresources
```

Comment:
```suggestion
  name: {{ include "insights-agent.fullname" $ }}-opa-defaulttargetresources
```

Reply:
`deft` confused me - maybe we can find another way to shorten? It's probably short enough though.

Reply:
I verified there isn't a 63 character name limit here (which we could be close to depending on the Helm release name), and changed these `deftargetresources` names to the full `defaulttargetresources` name.